airbyte-integrations/connectors/source-github/acceptance-test-config.yml

```diff
@@ -32,6 +32,11 @@ acceptance_tests:
       - name: "events"
         bypass_reason: "Only events created within the past 90 days can be showed"
       ignored_fields:
+        workflows:
+        - name: created_at
+          bypass_reason: "Got timezone +03:00 when expected +02:00"
```

Comment:
Why we get +03:00?

Reply:
It is important not to overlook any changes made to the field as it can result in a decrease in test coverage.
I request that you investigate this matter. It may be necessary to verify the time zone settings in the configuration and compare them with the actual summer time. Additionally, please consider updating the expected records with an appropriate justification.

Reply:
It is important not to overlook any changes made to the field as it can result in a decrease in test coverage.
I request that you investigate this matter. It may be necessary to verify the time zone settings in the configuration and compare them with the actual summer time. Additionally, please consider updating the expected records with an appropriate justification.